### ISSUE TITLE
appdata.xml contains invalid icon

### ISSUE THREAD

**Issue by dvzrv** [Yoshimi/yoshimi#172]
Hi! Since https://github.com/Yoshimi/yoshimi/commit/7607f90f1d0fb79604a2ebb15593c7bb0cb71eda the yoshimi.appdata.xml contains an icon.

Unfortunately this icon is not valid and is being skipped:

```
/usr/bin/appstreamcli refresh-cache --force
** (appstreamcli:159368): WARNING **: 10:55:46.224: Found icon of unknown type 'unknown' in 'system/package/os/net.sourceforge.projects.yoshimi.desktop/*', skipping it.
```

**Comment by abrolag** [comment]
On Fri, 27 Jan 2023 02:16:54 -0800
David Runge ***@***.***> wrote:

>Hi! Since https://github.com/Yoshimi/yoshimi/commit/7607f90f1d0fb79604a2ebb15593c7bb0cb71eda the yoshimi.appdata.xml contains an icon.
>
>Unfortunately this icon is not valid and is being skipped:
>
>```
>/usr/bin/appstreamcli refresh-cache --force
>** (appstreamcli:159368): WARNING **: 10:55:46.224: Found icon of unknown type 'unknown' in 'system/package/os/net.sourceforge.projects.yoshimi.desktop/*', skipping it.
>```
>

Thanks for the warning. I don't know how we picked up the '?' and digits, it's
supposed to be just:
https://a.fsdn.com/allura/p/yoshimi/icon

I'll correct it in the next push.

-- 
Will J Godfrey {apparently now an 'elderly'}

**Comment by abrolag** [comment]
Hopefully resolved from V 2.2.3 onwards

**Comment by cphil46** [comment]
Unfortunately not :
from apt : 
Paramétrage de appstream (0.16.1-2) ...

** (appstreamcli:304446): WARNING **: 20:43:03.953: Found icon of unknown type 'unknown' in 'system/package/os/net.sourceforge.projects.yoshimi.desktop/*', skipping it.
✔ Le cache des métadonnées a été mis à jour avec succès.

#yoshimi --help
Not there
Usage: yoshimi [OPTION...]
            Yoshimi 2.2.3, a derivative of ZynAddSubFX - Copyright 2002-2009

**Comment by abrolag** [comment]
On Tue, 30 May 2023 11:55:29 -0700
cphil46 ***@***.***> wrote:

>Unfortunately not :
>from apt : 
>Paramétrage de appstream (0.16.1-2) ...
>
>** (appstreamcli:304446): WARNING **: 20:43:03.953: Found icon of unknown type 'unknown' in 'system/package/os/net.sourceforge.projects.yoshimi.desktop/*', skipping it.
>✔ Le cache des métadonnées a été mis à jour avec succès.
>
>#yoshimi --help
>Not there
>Usage: yoshimi [OPTION...]
>            Yoshimi 2.2.3, a derivative of ZynAddSubFX - Copyright 2002-2009
>
>
Can you try the (just released) Yoshimi 2.3.0 ? We've made further changes.

-- 
Will J Godfrey {apparently now an 'elderly'}
https://willgodfrey.bandcamp.com/
http://yoshimi.github.io
Say you have a poem and I have a tune.
Exchange them and we can both have a poem, a tune, and a song.
